docs/apis/jsonrpc-apis.md

```diff
@@ -39,10 +42,11 @@ You'll need to use your own programming environments' capabilities (libraries, t
 connect via HTTP, WS or IPC to a `geth` node configured with the above flags and you'll
 need to speak [JSON-RPC](https://www.jsonrpc.org/specification) on all transports. You
 can reuse the same connection for multiple requests!
-
-**Note: Please understand the security implications of opening up an HTTP/WS based
-transport before doing so! Hackers on the internet are actively trying to subvert
-Ethereum nodes with exposed APIs! Further, all browser tabs can access locally
-running web servers, so malicious web pages could try to subvert locally available
-APIs!**
-
+Here you can check the available [JSON-RPC calls](https://playground.open-rpc.org/?schemaUrl={{ openrpc_latest_schema_endpoint }}){: target=_blank }.
```

Comment:
This is broken for me as Github renders it

Reply:
Good catch man. So using this I am able to set a config variable with the actual path. Though if we want (makes sense) to not break support for our users who read the docs folders or the MD files directly, then it doesn't worth using variables.

I vote for removing variables. Please let me know what you think.

Reply:
I agree... it's a nice-to-have convenience, but we can do almost the same thing with careful typing and find-and-replace :joy:

Reply:
Resolved by https://github.com/etclabscore/core-geth/pull/300/commits/ab98c44c5d9e3032fa40047f1e3cf3bde1f4827b